tools/lint.js

```diff
@@ -63,6 +51,50 @@ async function dlint() {
   }
 }
 
+async function dlintPreferPrimordials() {
+  const execPath = getPrebuiltToolPath("dlint");
+  console.log("prefer-primordials");
+
+  const sourceFiles = await getSources(ROOT_PATH, [
+    "runtime/**/*.js",
+    "ext/**/*.js",
+    "core/**/*.js",
+    ":!:core/examples/**",
+  ]);
+
+  if (!sourceFiles.length) {
+    return;
+  }
+
+  const chunks = splitToChunks(sourceFiles, `${execPath} run`.length);
+  for (const chunk of chunks) {
+    const p = Deno.run({
+      cmd: [execPath, "run", "--rule", "prefer-primordials", ...chunk],
+    });
+    const { success } = await p.status();
+    if (!success) {
+      throw new Error("prefer-primordials failed");
+    }
+    p.close();
+  }
+}
```

Comment:
I think `prefer-primordials` should apply only to the code related to bootstrapping, so I created another function where only this rule is run.

Reply:
Makes sense, please add a comment explaining that

Reply:
Done 👍  0cf3bfb